.NET/Microsoft.Recognizers.Text.DateTime/English/Parsers/EnglishTimeParserConfiguration.cs

```diff
@@ -114,18 +114,17 @@ public void AdjustBySuffix(string suffix, ref int hour, ref int min, ref bool ha
                         }
                     }
 
-                    var pmStr = match.Groups[Constants.PmGroupName].Value;
-                    if (!string.IsNullOrEmpty(pmStr))
+                    var time_pmStr = match.Groups[Constants.PmGroupName].Value;

```

Comment:
Isn't this a weird naming scheme?

Reply:
Yes, we are going to change it to `matchAmStr` to better describe the content of the variable.